### ISSUE TITLE
Feature Request - Identity command should have easy way of reporting balance of AGI and ETH

### ISSUE THREAD

**Issue by ferrouswheel** [singnet/snet-cli#11]
Checking on one's balance is probably a common enough use case that it'd be nice to make it easy for people via a single command.

Currently the two pieces of information can be obtained by the contract command:

```
snet contract SingularityNetToken balanceOf MY_ADDRESS
```

And the python code:
```
from web3 import Web3, HTTPProvider
w3 = Web3(HTTPProvider("https://kovan.infura.io"))
w3.eth.getBalance(MY_ADDRESS)
```

Both of these return large integers, without a decimal point delimiting the whole currency amount. ETH and AGI also have different decimal point positions.

To avoid making the user think, they should by default be converted to AGI and ETH. (e.g. instead of showing `2657719035000000000`, it should show `2.6577 ETH` or some other number of significant figures)

**Comment by arturgontijo** [comment]
Makes sense to me.
I also think that would be nice to have a command to ask for kAGI from faucet.
What do you guys think about it? @ferrouswheel, @vsbogd, @stellarspot

**Comment by vsbogd** [comment]
@arturgontijo, I am not sure I understand you correctly, do you mean some operation on AGI tokens balance in Kovan network? Which specific operation do you mean?

**Comment by arturgontijo** [comment]
Just a better way to call:
1. `snet contract SingularityNetToken balanceOf 0xFF2a327ed1Ca40CE93F116C5d6646b56991c0ddE'
2. `snet contract SingularityNetToken transfer 0x7fA3FBCFAc5c56367cB33821968Fc8c086199989 10`
3. `snet contract SingularityNetToken transferTokens 0x7fA3FBCFAc5c56367cB33821968Fc8c086199989 10`

I'm doing these inside a new `snet wallet` (what do you think?) command:
1. `snet wallet balance`
```
    identity:
        name: ARTUR_ID
        network: '42'
        wallet: '0xFF2a327ed1Ca40CE93F116C5d6646b56991c0ddE'
    wallet:
        AGI: 5.77382734
        ETH: 3.199385969513249
```
2. `snet wallet transfer 0x7fA3FBCFAc5c56367cB33821968Fc8c086199989 10`
2. `snet wallet transferTokens 0x7fA3FBCFAc5c56367cB33821968Fc8c086199989 10`

**Comment by pennachin** [comment]
I'm OK with reads. I don't want to get into the business of writing an API to ETH wallets and the implications of that.

**Comment by astroseger** [comment]
will be solved by #109.

For example "snet client balance" will print your balance of AGI tokens and balance inside mpe_wallet 
``` 
$ snet client balance 
agi_balance: '1000000000'
mpe_wallet_balance: '0'
```

**Comment by ferrouswheel** [comment]
Note this issue is also asking to show ETH, and that the balance for both be shown as a user-friendly value. I wouldn't consider this issue solved by #109 as it stands.

Not the raw integer which is not a pleasant user experience.

If it *must* show the raw value, then we should at least be explicit and use the unit of "cogs" instead of "agi"

**Comment by astroseger** [comment]
1.  "snet client balance" do not show raw integer (not cogs!). It shows AGI. In general in new client and server, user always input AGI, and always sees AGI. 
2. I've added ETH balance. 
3. I've added possibility to print balance of any account (snet client balance --account <account_to_print_balance_for>)

**Comment by ferrouswheel** [comment]
Oh, awesome! You have a lot of AGI compared to me ;-) Thanks Sergey

**Comment by astroseger** [comment]
was solved by #109
